browser-test/src/applicant/northstar_ineligible.test.ts

```diff
@@ -189,6 +189,7 @@ test.describe('North Star Ineligible Page Tests', {tag: ['@northstar']}, () => {
       // Click "Continue" on the program card
       await applicantQuestions.clickApplyProgramButton(programName)
 
+      await applicantQuestions.clickEdit()
```

Comment:
**question (non-blocking):** I don't understand how this test is passing.  Haven't you already clicked on the program card "Edit" button before that in `applicantQuestions.clickApplyProgramButton`?  
![image](https://github.com/user-attachments/assets/9632faed-e381-4ca4-b78f-35a56cd93244)
The "Edit" button meets the requirements of that locator, no?

Reply:
So much easier to understand.  Thanks a bunch!